client/src/components/Header/Header.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import Header from './Header'
+
+describe('Header', () => {
+  it('renders without crashing', () => {
+    const { container } = render(<Header title='title' subtitle='subtitle' />)
+    expect(container.firstElementChild).toBeInTheDocument()
+  })
+
+  it('renders a Header', () => {
+    const { container } = render(<Header title='title' subtitle='subtitle' />)
+    expect(container.firstElementChild!.tagName).toBe('HEADER')
+  })
+
+  it('renders the title prop', () => {
+    const { getByText } = render(<Header title='title' subtitle='subtitle' />)
+    const title = getByText('title')
+    expect(title).toHaveTextContent('title')
```

Comment:
It does not make sense now to find an element by some text, then test that it indeed contains the text.

Reply:
Maybe we could find the title element by its semantic role (`heading`) and verify that it has the expected text content 🤔

Reply:
Yep indeed. Added the changes:
- ByRole for the heading.
- QuerySelector for the subtitle because a `span` has no role (check my question in #front in Slack about this).